torch/csrc/jit/python/pybind_utils.h

```diff
@@ -703,7 +703,6 @@ inline void guardAgainstNamedTensor(const T& var) {
 
 // Defined in pybind_utils.cpp to break a circular dependency with
 // python_ivalue.h
```

Comment:
If above comment is no longer relevant, please remove it as well

Reply:
Another commit removes the comment